### PR TITLE
Fixing grammar and spelling in chapter 1

### PR DESCRIPTION
**Pull request by natcon0368**

### CODE REVIEW

File: pages/acknowledgments.tex [hunk 1]
```diff
@@ -8,16 +8,16 @@
 
 
 
-\section*{About this rulebook}
-This is the official rulebook of the RoboCup@Home competition \YEAR.
+\section*{About This Rulebook}
```
Comment (LoyVanBeek) [inline pages/acknowledgments.tex]: Should `This` really be capitalized?

File: pages/acknowledgments.tex [hunk 2]
```diff
@@ -8,16 +8,16 @@
...
+This is the official rulebook of the \YEAR ~RoboCup@Home competition.
 It has been written by the \YEAR ~RoboCup@Home Technical Committee with the special collaboration of (in alphabetical order):
 % Mauricio Matamoros,
 % and
 % Loy van Beek.
 
 
 
-\section*{How to cite this rulebook}
+\section*{How to Cite This Rulebook}
```
Comment (LoyVanBeek) [inline pages/acknowledgments.tex]: `Cite` and `This` with capitals?

Comment (kyordhel) [reply pages/acknowledgments.tex]: AFAIK in English it is good practice to capitalize every word but pronouns in titles. *This* is a determiner, so according to some styles it gets capitalized.

Other conventions are more flexible and only require words of 4+ characters to be capitalized unless they are verbs, adjectives, pronouns, or nouns.

Maybe @justinhart can explain us further

File: pages/acknowledgments.tex (hunk 1 above)
Comment (maximest-pierre) [reply pages/acknowledgments.tex]: So after looking at the dictionary :book: this should not be capitalise cause its not a major word. Anything like this, the, or, and should not be capitalise.

File: pages/acknowledgments.tex (hunk 2 above)
Comment (maximest-pierre) [reply pages/acknowledgments.tex]: {How to Cite this Rulebook} would be the appropriate spelling.

File: pages/acknowledgments.tex (hunk 1 above)
Comment (moriarty) [reply pages/acknowledgments.tex]:
```suggestion
\section*{About this Rulebook}
```

File: pages/acknowledgments.tex (hunk 2 above)
Comment (moriarty) [reply pages/acknowledgments.tex]:
```suggestion
\section*{How to Cite this Rulebook}
```

**Comment by moriarty** [comment]
This PR is getting old, @maximest-pierre and @LoyVanBeek I've used the GitHub UI to suggest changes, this makes it easier for the authors to apply feedback.

There are also conflicts now, if the GitHub UI is used to fix them it merges the master branch into this PR branch... So I suggest squashing and merging if this once it is ready to go in.

**Comment by johaq** [comment]
Obsolete with #722?
I'll close this for now. Most should be fixed with new PR.

**Pull request closed without merging**